godtoolsTests/App/Features/AppLanguage/Data-DomainInterface/GetLanguageSettingsInterfaceStringsRepositoryTests.swift

```diff
@@ -6,21 +6,172 @@
 //  Copyright © 2024 Cru. All rights reserved.
 //
 
-import Foundation
+import Testing
 @testable import godtools
 import Combine
-import Quick
-import Nimble
 
-class GetLanguageSettingsInterfaceStringsRepositoryTests: QuickSpec {
+struct GetLanguageSettingsInterfaceStringsRepositoryTests {
     
-    override class func spec() {
+    @Test(
+        """
+        Given: User is viewing the language settings.
+        When: The app is switched from English to Spanish.
+        Then: The interface strings should be translated into Spanish.
+        """
+    )
+    @MainActor func interfaceStringsAreTranslatedWhenAppLanguageChanges() async {
```

Comment:
Hey @rachaelblue because the TestsRealmDatabase by default uses a single realm instance on the main thread, there will be a crash when the realm instance is observing collection changes.
```*** Terminating app due to uncaught exception 'RLMException', reason: 'Can only add notification blocks from within runloops.'
terminating due to uncaught exception of type NSException```

To ensure there isn't a crash the Test will need to be marked with the ```MainActor``` to keep execution on the main thread.

Reply:
Thanks @levieggertcru!